usaspending_api/common/management/commands/matview_runner.py

```diff
@@ -13,6 +13,7 @@
     DEPENDENCY_FILEPATH,
     DROP_OLD_MATVIEWS,
     MATERIALIZED_VIEWS,
+    CHUNKED_MATERIALIZED_VIEWS,
```

Comment:
Minor detail, these imports were listed alphabetically. Not all in the codebase are, but it is moving in that direction